js/src/scrollspy.js

```diff
@@ -269,7 +269,7 @@ const ScrollSpy = (($) => {
     static _jQueryInterface(config) {
       return this.each(function () {
         let data    = $(this).data(DATA_KEY)
-        let _config = typeof config === 'object' && config || null
```

Comment:
This is not the same thing: if config is `undefined` this will evaluate to `undefined` where the previous version would evaluate to `null`.